tests/unit/graph_reader/test_artifact_collection.py

```diff
@@ -77,7 +77,9 @@ def test_one_session(execute, input_script, artifact_list, expected_output):
     artifact_string = ", ".join([f'"{x}"' for x in artifact_list])
     code = (
         "from lineapy.graph_reader.artifact_collection import ArtifactCollection\n"
-        + f"ac = ArtifactCollection([{artifact_string}])"
+        + "from lineapy.execution.context import get_context\n"
+        + "db = get_context().executor.db\n"
+        + f"ac = ArtifactCollection(db, [{artifact_string}])"
     )
     res = execute(code, snapshot=False)
     ac = res.values["ac"]
```

Comment:
you probably dont need the 2nd execute call. the artifactcollection can be initialized directly for the sake of this test. and use the instance for subsequent asserts.

Reply:
Since we have other similar parts in codebase, I will address this fix in a separate PR.